infra/core/observability/azure-monitor-workspace.bicep

```diff
@@ -0,0 +1,26 @@
+@description('Required. The Name of the Azure Monitor Workspace.')
+param azureMonitorWorkspaceName string
+
+@description('Required. The Azure region where the resources will be deployed.')
+param location string
+
+@description('Required. Environment name.')
+param environment string
+
+@description('Optional. Date in the format yyyy-MM-dd.')
+param createdDate string = utcNow('yyyy-MM-dd')
+
+var commonTags = {
+  Name: azureMonitorWorkspaceName
+  Location: location
+  CreatedDate: createdDate
+  Environment: environment
+  Purpose: 'Azure Monitor Workspace'
```

Comment:
Purpose: Shared Azure Monitor Workspace for Platform

Reply:
Updated